controllers/render_template.go

```diff
@@ -96,3 +115,24 @@ func applySizeTemplate(cs *unstructured.Unstructured, sizeTemplate string) ([]in
 	}
 	return sizes, nil
 }
+
+func (r *CommonServiceReconciler) checkSaas() (enable bool, err error) {
+	cmName := constant.SaasConfigMap
+	cmNs := "kube-public"
+	saasConfigmap := &corev1.ConfigMap{}
+	err = r.Reader.Get(context.TODO(), types.NamespacedName{Name: cmName, Namespace: cmNs}, saasConfigmap)
+	if errors.IsNotFound(err) {
+		return false, nil
+	} else if err != nil {
+		return false, err
+	}
+	v, ok := saasConfigmap.Data["ibm_cloud_saas"]
+	if !ok {
+		return false, nil
+	}
+	if v != "true" {
+		klog.Info(v)
```

Comment:
is the `klog.Info(v)` too unclear as a information message, i assume that `v` is just `false`?

Reply:
Sorry, it is for testing. I forget to remove it.